test/utils-liquity.js

```diff
@@ -0,0 +1,27 @@
+const hre = require('hardhat');
+const { getAddrFromRegistry } = require('./utils');
+
+const LiquityActionIds = {
+    Open: 0,
+    Borrow: 1,
+    Payback: 2,
+    Supply: 3,
+    Withdraw: 4,
+};
+
+const getHints = async (troveOwner, actionId, from, collAmount, LUSDamount) => {
+    const liquityView = await hre.ethers.getContractAt('LiquityView', getAddrFromRegistry('LiquityView'));
+
+    const NICR = await liquityView['predictNICR(address,uint8,address,uint256,uint256)'](troveOwner, actionId, from, collAmount, LUSDamount);
```

Comment:
Couldn't this be one call in a function that groups all 3 methods on the View contract?